src/helpers/createChart.ts

```diff
@@ -1,27 +1,25 @@
-const fs = require('fs');
-const { JSDOM } = require('jsdom');
-const d3 = require('d3');
+import fs from 'fs';
+import { JSDOM } from 'jsdom';
+import * as d3 from 'd3';
 
 /**
  * Output memory usage data to SVG line graph
- *
- * @param {Object} data
- * @param {String} path
  */
-module.exports = (data, path) => {
+export default (data: Array<Array<number>>, path: string): void => {
   const dom = new JSDOM('<html><body></body></html>');
   const margin = {
-    top: 20, right: 20, bottom: 30, left: 60
+    top: 20, right: 20, bottom: 30, left: 60,
   };
   const w = 900 - margin.left - margin.right;
   const h = 500 - margin.top - margin.bottom;
 
-  let svg = d3.select(dom.window.document).select('body')
+  let svg: any = d3.select(dom.window.document).select('body')
     .append('svg')
     .attr('width', w + margin.left + margin.right)
     .attr('height', h + margin.top + margin.bottom);
 
   // Background cheat
+  /* eslint-disable */
```

Comment:
Why not just work with the typesystem and not against.  `svg` does have a defined type. Besides, `never` or `unknown` work better when you are working with generics.